file_paths.md

```diff
@@ -163,156 +163,167 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
     Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    C:\ProgramData\PuppetLabs *                                              n/a
 
     C:\ProgramData\PuppetLabs\client-tools    # default client tool settings
         puppet-access.conf *
         puppet-orchestrator.conf *
 
-
     C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
         environments *                        # :environmentpath
           production *
             environment.conf *
+            hieradata *
             manifests *
             modules *
-        hiera.yaml *                          # :hiera_config
-        hieradata *                           # n/a
-        modules *                             # user modulepath
+        hiera.yaml                            # :hiera_config
+        modules                               # user modulepath
 
-    C:\ProgramData\PuppetLabs\facter *                                       same
+    C:\ProgramData\PuppetLabs\mcollective *
```

Comment:
They are created in ProgramData if they do not exist, but otherwise they are completely left alone.